sql/moz-fx-data-shared-prod/mozilla_vpn_derived/users_v1/schema.yaml

```diff
@@ -8,28 +8,3 @@ fields:
 - mode: NULLABLE
   name: created_at
   type: TIMESTAMP
-- mode: NULLABLE
-  name: attribution
```

Comment:
Attribution is being dropped in general in the users table?

Reply:
yep